core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -190,10 +200,9 @@ object ConfigCommand {
           throw new IllegalArgumentException("At least one of --add-config or --delete-config must be specified with --alter")
       }
       val entityTypeVal = options.valueOf(entityType)
-      if(! entityTypeVal.equals(ConfigType.Topic) && ! entityTypeVal.equals(ConfigType.Client)) {
-        throw new IllegalArgumentException("--entity-type must be '%s' or '%s'".format(ConfigType.Topic, ConfigType.Client))
+      if(! entityTypeVal.equals(ConfigType.Topic) && ! entityTypeVal.equals(ConfigType.Client) && ! entityTypeVal.equals(ConfigType.Broker)) {
+        throw new IllegalArgumentException("--entity-type must be '%s' or '%s' or '%s'".format(ConfigType.Topic, ConfigType.Client, ConfigType.Broker))
```

Comment:
Seems like this code would be a bit nicer if we had a `val supportedTypes = Set(ConfigType.Topic, ConfigType.Client, ConfigType.Broker)` somewhere.